src/seed/unseed-db.js

```diff
@@ -0,0 +1,27 @@
+import { query } from '../graphql-schema'
+import { v1 as neo4j } from 'neo4j-driver'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const driver = neo4j.driver(
```

Comment:
We shoud use the driver from a central location instead of initialization it again here.

Reply:
Yes right, I already though about it